oneandone/src/test/java/org/apache/jclouds/oneandone/rest/internal/BaseOneAndOneLiveTest.java

```diff
@@ -143,4 +143,17 @@ protected Server turnOnServer(String serverId) {
    protected Server turnOFFServer(String serverId) {
       return api.serverApi().updateStatus(serverId, Server.UpdateStatus.create(Types.ServerAction.POWER_OFF, Types.ServerActionMethod.SOFTWARE));
    }
+
+   protected Server GetRandomServerWithMonitoringPolicy() throws InterruptedException {
+      List<Server> result = api.serverApi().list();
+      for (Server server : result) {
+         Thread.sleep(2000);
+         server = api.serverApi().get(server.id());
+         if (server.monitoringPolicy() != null) {
+            return server;
+         }
+      }
+
+      return null;
```

Comment:
Creating a server is not enough since i will need to create a monitoring policy on that server too, that is the next PR, maybe open the next PR and then merge it into this one to be able to create the policy, i am following the order of the API documentation for the PR'S WDYT?